### ISSUE TITLE
Super slow builds using ember with ember-auto-import

### ISSUE THREAD

**Issue by nerdybeast** [ember-cli/ember-cli#8472]
Thank you for taking the time to open an issue!

For bug reports please include the following section in your issue details.
If you include instructions on how to reproduce the bug or a failing test case
it will make it easier for us to track down the issue you're having.

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
λ ember -v --verbose
ember-cli: 3.8.1
http_parser: 2.8.0
node: 10.15.1
v8: 6.8.275.32-node.12
uv: 1.23.2
zlib: 1.2.11
ares: 1.15.0
modules: 64
nghttp2: 1.34.0
napi: 3
openssl: 1.1.0j
icu: 62.1
unicode: 11.0
cldr: 33.1
tz: 2018e
os: win32 x64

λ npm --version
6.8.0
```

Not sure what's going on here but using ember-auto-import with any ember version higher than 3.3.0 is causing insanely long build times. The reason I think this is an issue with ember-cli is because if I use ember version 3.3.0, everything builds quickly, as soon as I upgrade (using ember-cli-update) to version 3.4+ the slow build times occur.

I am developing on Windows and have checked all the boxes for development:

- Running terminal as administrator 
- Ran ember-cli-windows (which doesn't do anything because Windows Defender is disabled on this computer)
- Made sure I have symbolic link permissions

I have created an example repo that reproduces the issue: https://github.com/nerdybeast/slow-ember-build.git

I simply created a new project (ember version 3.8.X)
Ran `ember install ember-auto-import`
npm installed some dependencies and added them as imports into the `ember-cli-build.js` file.

Before adding any npm dependencies, this app builds in a few seconds but as soon as I import those references into the build file, the build takes 2+ minutes and any file change takes that long as well.

![screenshot_66](https://user-images.githubusercontent.com/15707802/53700116-971a4680-3db4-11e9-9c10-ff9f98073f5a.png)

**Comment by rwjblue** [comment]
Hmm, its not immediately obvious that this is an issue with ember-cli vs ember-auto-import. Did you happen to open an issue up over there also?

**Comment by stefanpenner** [comment]
Ya, if the issue began when using `ember-auto-import` which is a user-land add-on, the issue should be opened there instead: https://github.com/ef4/ember-auto-import

**Comment by stefanpenner** [comment]
Please open the issue their, but feel free to cross link.

**Comment by ef4** [comment]
The reproduction repo that was shared *is just as slow with ember-auto-import removed*.

The trigger for the slowness in that app isn't an auto-import, it's an `app.import` of a font from a huge NPM package.

**Comment by nerdybeast** [comment]
If I downgrade to ember version 3.3.0 with this same setup it builds reasonably fast so I'm pretty sure that changes in 3.4+ have caused the slower build to occur.

**Comment by stefanpenner** [comment]
can you share an example that does `app.import` + that large npm module?
At one point, I fast pathed these scenarios. Maybe some subsequent work broke it?

A reproduction would help diagnose quickly.

**Comment by nerdybeast** [comment]
@stefanpenner I created this repo that reproduces the issue: https://github.com/nerdybeast/slow-ember-build.git

It's importing a few node modules including the large font file (from materialize)

**Comment by stefanpenner** [comment]
Interesting, I even remember fixing the materialize use-case years ago. This is most likely a regression...

**Comment by nerdybeast** [comment]
Well apparently doing
`
app.import('node_modules/material-design-icons/iconfont/MaterialIcons-Regular.eot');
app.import('node_modules/material-design-icons/iconfont/MaterialIcons-Regular.svg');
app.import('node_modules/material-design-icons/iconfont/MaterialIcons-Regular.ttf');
app.import('node_modules/material-design-icons/iconfont/MaterialIcons-Regular.woff');
`
is not the right way to load font files anyway (which is what I was doing). If I just manually placed them in `public/assets` the build is significantly faster. This is according to the new docs: https://cli.emberjs.com/release/advanced-use/asset-compilation/#rawassets

I was doing it based on the instructions here: https://ember-cli.com/user-guide/#other-assets

Perhaps this is an edge case because the size of the material design icon files but I would imagine that other fonts out there are equally as big.

Either way, I have found a work around that is easy enough to do. Thanks for looking into this!
